webpack/assets/javascripts/react_app/components/notifications/notifications.scss

```diff
@@ -3,6 +3,8 @@
 #notifications_container {
   position: static;
   margin-top: -1px;
+  color: var(--pf-global--palette--black-1000);
```

Comment:
I think you can also use something like: `$--pf-global--palette--black-1000`
but maybe you need to import it correctly from `~patternfly`

Reply:
True.
IMO it is better to use the css variables as they provide more value (change in real-time, supported by the browser).
Also, looks like pf4 are transitioning from scss variables to css variables and it might be wise to embrace it.

Reply:
I didn't find it in the PF docs, can you point it out to me? :)